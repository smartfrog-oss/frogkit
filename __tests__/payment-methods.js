import { mount } from 'vue-test-utils'
import PaymentMethods from '@/components/payment-methods'

describe('Payment Methods component', () => {

  it('should render component and match snapshot', () => {
    const wrapper = mount(PaymentMethods)
    expect(wrapper.html()).toMatchSnapshot()
  })

})
