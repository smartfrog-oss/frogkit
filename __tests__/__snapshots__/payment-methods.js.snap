// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Payment Methods component should render component and match snapshot 1`] = `
<section class="payment-methods">
  <flex grow="" align="center" justify="space-around" class="m-b-30"><img src="../assets/payment/visa.svg" alt="visa" class="payment-methods__visa"> <img src="../assets/payment/mastercard.svg" alt="mastercard" class="payment-methods__masterCard"> <img src="../assets/payment/american_express.svg" alt="american express" class="payment-methods__americanExpress"> <img src="../assets/payment/sepa.svg" alt="sepa" class="payment-methods__sepa"></flex>
  <flex grow="" align="center" justify="space-around"><img src="../assets/payment/klarna.svg" alt="klarna" class="payment-methods__klarna"> <img src="../assets/payment/ec.svg" alt="ec" class="payment-methods__ec"> <img src="../assets/payment/paypal.svg" alt="paypal" class="payment-methods__paypal"></flex>
</section>
`;
